engine/src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -83,6 +84,35 @@ class CromwellApiHandler(requestHandlerActor: ActorRef) extends Actor {
 
   private def error(t: Throwable)(f: Throwable => RequestComplete[_]): Unit = context.parent ! f(t)
 
+  //Use Uri.withQuery to ensure proper uri encoding.
+  private def generatePaginationParams(page: Int, pageSize: Int): String = {
+    s"?page=$page&pagesize=$pageSize"
```

Comment:
I think the `?` is not necessary here. I suspect that the `withQuery` function injects the `?`. The links are currently coming back as `Link:<http://localhost:8080/api/workflows/v1/query??page=1&pagesize=5>; rel=first`